arduino_output/OdomRaw.msg

```diff
@@ -0,0 +1,11 @@
+# Message to transport raw odometry information from
+# the Arduino microcontroller to the main ROS system.
+# It is expected that this raw data will be expanded
+# into the full odometry messages used by ROS.
+
+Header header
+float64 x
```

Comment:
Pose contains a Point and Quaternian. Twist has two Vector3's. It is unclear to me that we are saving any space by using those compared to 5 float64's.